### DISCUSSION TITLE
CDN url

### DISCUSSION THREAD

**Discussion by Lukasz93P** [livewire/livewire#3928] [7. Bugs (archived)] [answered]
Hi @calebporzio I don't understand why this has been closed by you, this is definitely a bug in my opinion.
https://github.com/livewire/livewire/issues/2548

**Comment by joshhanley** [comment, marked as answer]
@Lukasz93P it hasn’t been closed, it’s actually been converted to a discussion, see #3108. 

The reason this issue had been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

Hope this helps!

**Comment by Lukasz93P** [comment]
@joshhanley  ok, I will provide pull request with my solution on that.

**Comment by Lukasz93P** [comment]
@joshhanley here it is https://github.com/livewire/livewire/pull/3929/commits
